components/plugin-manager/api/build.gradle.kts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2025 The ORT Server Authors (See <https://github.com/eclipse-apoapsis/ort-server/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+plugins {
+    id("ort-server-kotlin-multiplatform-conventions")
+    id("ort-server-publication-conventions")
+
+    // Apply third-party plugins.
+    alias(libs.plugins.kotlinSerialization)
+}
+
+group = "org.eclipse.apoapsis.ortserver.components.pluginmanager"
```

Comment:
I am considering to rename the `api` and `implementation` modules to `frontend` and `backend` in a follow-up to clarify their purpose, please share your opinion.

Reply:
Isn't "frontend" usually the UI? Also I'm used to the "api" and "implementation" terms from Gradle dependencies. So if this is about a similar concept on the module level, I think I'd prefer to stick with "api" and "implementation".

Reply:
This is the misunderstanding I was thinking of, "api" actually contains models for the REST API and therefore needs to be a multiplatform module, "implementation" contains the whole backend implementation including the JVM API to be consumed by other modules.
But yes, "frontend" sounds like it would be the frontend, and "api" is already used for the top-level module, so maybe "api" and "backend"?

Reply:
I am also fine with the terms "api" and "implementation". I would prefer "implementation" over "backend" because of the risk of confusing this with the distinction between frontend and backend.